### PR TITLE
R2: add egress == free to table

### PR DESCRIPTION
**Pull request by elithrar**
cc @jonesphillip

### CODE REVIEW

File: content/r2/pricing.md [hunk 1]
```diff
@@ -24,9 +24,12 @@ All included usage is on a monthly basis.
 | Storage            | 10 GB / month                | $0.015 / GB-month                  |
 | Class A Operations | 1 million requests / month   | $4.50 / million requests           |
 | Class B Operations | 10 million requests / month  | $0.36 / million requests           |
+| Egress (data transfer to Internet) | Free <sup>1</sup> |
 
 {{</table-wrap>}}
 
+<sup>1</sup> Egressing directly from R2, including via the Workers API, S3 API, and `r2.dev` domains does not incur data transfer (egress) charges and is free. If you connect other, metered services to an R2 bucket, you may be charged by those services.
```
Comment (kodster28) [inline content/r2/pricing.md]:
```suggestion
<sup>1</sup> Egressing directly from R2, including via the [Workers API](/r2/api/workers/), [S3 API](/r2/api/s3/), and [`r2.dev` domains](/r2/buckets/public-buckets/) does not incur data transfer (egress) charges and is free. If you connect other, metered services to an R2 bucket, you may be charged by those services.
```

**Review by kodster28 — APPROVED** [review]

File: content/r2/pricing.md (hunk 1 above)
Comment (kodster28) [inline content/r2/pricing.md]: Also, @elithrar, might just be your preference, but we do have some fancier footnote formatting available through [`littlefoot.js`](https://littlefoot.js.org/) that overwrites default markdown footnotes.

[Example behavior](https://developers.cloudflare.com/cloudflare-for-platforms/cloudflare-for-saas/start/advanced-settings/apex-proxying/#benefits), the 3 dots after `at the zone's root`.

Used to be an issue occasionally around the width of the footnote expansion, but believe that's fixed now.

Comment (elithrar) [reply content/r2/pricing.md]: I definitely prefer the little foot approach: would you mind suggesting / adding it @kodster28 - ?

(Traveling and only have my phone, so PR'ing things is trickier than usual)

File: content/r2/pricing.md [hunk 2]
```diff
@@ -24,9 +24,12 @@ All included usage is on a monthly basis.
 | Storage            | 10 GB / month                | $0.015 / GB-month                  |
 | Class A Operations | 1 million requests / month   | $4.50 / million requests           |
 | Class B Operations | 10 million requests / month  | $0.36 / million requests           |
+| Egress (data transfer to Internet) | Free <sup>1</sup> |
```
Comment (elithrar) [inline content/r2/pricing.md]:
```suggestion
| Egress (data transfer to Internet) | Free [^1] |
```

File: content/r2/pricing.md [hunk 3]
```diff
@@ -24,9 +24,12 @@ All included usage is on a monthly basis.
...
 
 {{</table-wrap>}}
 
+<sup>1</sup> Egressing directly from R2, including via the [Workers API](/r2/api/workers/), [S3 API](/r2/api/s3/), and [`r2.dev` domains](/r2/buckets/public-buckets/) does not incur data transfer (egress) charges and is free. If you connect other, metered services to an R2 bucket, you may be charged by those services.
```
Comment (elithrar) [inline content/r2/pricing.md]:
```suggestion
[^1]: Egressing directly from R2, including via the [Workers API](/r2/api/workers/), [S3 API](/r2/api/s3/), and [`r2.dev` domains](/r2/buckets/public-buckets/) does not incur data transfer (egress) charges and is free. If you connect other, metered services to an R2 bucket, you may be charged by those services.
```

File: content/r2/pricing.md [hunk 4]
```diff
@@ -24,9 +24,12 @@ All included usage is on a monthly basis.
 | Storage            | 10 GB / month                | $0.015 / GB-month                  |
 | Class A Operations | 1 million requests / month   | $4.50 / million requests           |
 | Class B Operations | 10 million requests / month  | $0.36 / million requests           |
+| Egress (data transfer to Internet) | Free [^1] |
```
Comment (elithrar) [inline content/r2/pricing.md:25]:
```suggestion
| Egress (data transfer to Internet) | Free[^1] |
```

File: content/r2/pricing.md [hunk 5]
```diff
@@ -24,9 +24,12 @@ All included usage is on a monthly basis.
 | Storage            | 10 GB / month                | $0.015 / GB-month                  |
 | Class A Operations | 1 million requests / month   | $4.50 / million requests           |
 | Class B Operations | 10 million requests / month  | $0.36 / million requests           |
+| Egress (data transfer to Internet) | Free[^1] |
```
Comment (elithrar) [inline content/r2/pricing.md]: @kodster28 this doesn’t seem to work inside a table. 

The footnote itself is hidden but the ellipses are never rendered.

**Pull request merged**